docs/sprint-goals.md

```diff
@@ -1,5 +1,14 @@
 # Notify Sprint Goals Log
 
+## Sprint: K (5/24/23)
+
+|             | Goals                                                                                                                              | Impact                                                                                                                                     |
+|-------------|-----------------------------------------------------------------------------------------------------------------------------------|--------------------------------------------------------------------------------------------------------------------------------------------|
+| Engineering | Moving from an ad-hoc "complete must-haves" process to a more strategic, themed progression; examining and implementing a structure for [Architecture Decison Records](https://github.com/orgs/GSA/projects/20/views/1?pane=issue&itemId=29045425); assessing and planning for timezone accommodations in both [back-end](https://github.com/orgs/GSA/projects/20/views/1?pane=issue&itemId=26823949) and [UI](https://github.com/orgs/GSA/projects/20/views/1?pane=issue&itemId=27984305)                                                          |   Less context-switching; better documentation and prioritization; a clear strategy and content regarding timezones for users                                   |
+| UX          | Wrap first iteration of [USWDS component migration](https://github.com/orgs/GSA/projects/20/views/1?pane=issue&itemId=25381437); [propose new flows and wireframes](https://github.com/orgs/GSA/projects/20/views/1?pane=issue&itemId=25381587)             | A U.S. government look and feel; easier user navigation          |
+| Content     | [Hone content](https://github.com/orgs/GSA/projects/20/views/1?pane=issue&itemId=29029519) for [U.S. Notify-specific aspects](https://github.com/orgs/GSA/projects/20/views/1?pane=issue&itemId=28956821); finalize [style and terminology docs](https://github.com/orgs/GSA/projects/20/views/1?pane=issue&itemId=26787564) | Clear and applicable documentation and direction in the application |
+| Security    | Respond to any assessment questions and/or needs                                                  | Keep the LATO award timeline as short as possible
+
```

Comment:
I noticed in the previous sprint goal updates that the links go directly to the issues as opposed to linking to the issue pane on the project board, which I think is easier to work with.  I have updated the links below to point directly to the issues, but if you intended to link them in this fashion to the project board then please feel free to ignore!  Though there is a spelling correction in there, too, but I can pull that out. :-)

```suggestion
| Engineering | Moving from an ad-hoc "complete must-haves" process to a more strategic, themed progression; examining and implementing a structure for [Architecture Decision Records](https://github.com/GSA/notifications-admin/issues/522); assessing and planning for timezone accommodations in both [back-end](https://github.com/GSA/notifications-api/issues/260) and [UI](https://github.com/GSA/notifications-admin/issues/525)                                                          |   Less context-switching; better documentation and prioritization; a clear strategy and content regarding timezones for users                                   |
| UX          | Wrap first iteration of [USWDS component migration](https://github.com/GSA/notifications-admin/issues/453); [propose new flows and wireframes](https://github.com/GSA/notifications-admin/issues/455)             | A U.S. government look and feel; easier user navigation          |
| Content     | [Hone content](https://github.com/GSA/notifications-admin/issues/523) for [U.S. Notify-specific aspects](https://github.com/GSA/notifications-admin/issues/521); finalize [style and terminology docs][(https://github.com/orgs/GSA/projects/20/views/1?pane=issue&itemId=26787564)](https://github.com/GSA/notifications-admin/issues/481) | Clear and applicable documentation and direction in the application |
| Security    | Respond to any assessment questions and/or needs                                                  | Keep the LATO award timeline as short as possible

```


<details>
<summary>Just in case you weren't sure how to quickly/easily grab the links directly to the issues themselves from the project board, click here for some tips!</summary>

All I did was go to the issue pane, then right-clicked on the issue title to bring up the context menu:

<img width="1488" alt="Screen Shot 2023-05-25 at 10 38 20 AM" src="https://github.com/GSA/notifications-admin/assets/121565/b8f396a9-c56f-4bf4-ab0b-869f1351a4ca">


Then I clicked on `Copy Link Address` in the menu to get the link to the issue itself:

<img width="674" alt="Screen Shot 2023-05-25 at 10 38 32 AM" src="https://github.com/GSA/notifications-admin/assets/121565/68c21990-2691-4050-8521-210cd4a474d2">

You can also do the same directly from the project board itself when you right-click to get the context menu on the issue title itself:

<img width="667" alt="Screen Shot 2023-05-25 at 10 43 44 AM" src="https://github.com/GSA/notifications-admin/assets/121565/99fde958-86e1-4fbd-b56c-d3e527db7f7e">
</details>

Reply:
All good by me! Thanks for the changes

Reply:
For links, if going directly to issue, I have previously clicked in the right lower nav of the card in the board. Good to know the copy link from the right click goes there, too. 
![image](https://github.com/GSA/notifications-admin/assets/6556888/a7adc8e6-4636-494e-a5b7-4096e95d15c2)